tests/dynamic_api_test.py

```diff
@@ -621,6 +621,14 @@ def test_flattening_basic(self):
     jaxpr = jax.make_jaxpr(lambda x: x.reshape(-1, 12), abstracted_axes={0: 'n'})(x)
     self.assertLessEqual(len(jaxpr.jaxpr.eqns), 3)
 
+  def test_shape_validation(self):
```

Comment:
This does not test the dynamic shapes message? Maybe add a test for that too?

Reply:
These are both dynamic shapes messages, because the shape argument is passed via `jax.make_jaxpr`

Reply:
Ohh cool. I misread sorry.